appengine/standard/firebase/firetactoe/static/main.js

```diff
@@ -29,7 +29,9 @@
 function initGame(gameKey, me, token, channelId, initialMessage) {
   var state = {
     gameKey: gameKey,
-    me: me
+    me: me,
+    channelId: channelId,
```

Comment:
Any particular reason to save the channelId into the state? It doesn't seem to be used..